src/Documentation/Advanced-Concepts/Docker-Deployment.md

```diff
@@ -0,0 +1,615 @@
+Creating orleansdocker_orleans-silo_14 ... 
+Creating orleansdocker_orleans-silo_15 ... 
+Creating orleansdocker_orleans-silo_6
+Creating orleansdocker_orleans-silo_5
+Creating orleansdocker_orleans-silo_3
+Creating orleansdocker_orleans-silo_2
+Creating orleansdocker_orleans-silo_4
+Creating orleansdocker_orleans-silo_9
+Creating orleansdocker_orleans-silo_7
+Creating orleansdocker_orleans-silo_8
+Creating orleansdocker_orleans-silo_10
+Creating orleansdocker_orleans-silo_11
+Creating orleansdocker_orleans-silo_15
+Creating orleansdocker_orleans-silo_12
+Creating orleansdocker_orleans-silo_14
+Creating orleansdocker_orleans-silo_13
+```
+
+Few seconds later, you will see the services scaled to the specific number of instances you requested.
+
+```shell
+# docker-compose ps
+             Name                     Command        State   Ports 
+------------------------------------------------------------------
+orleansdocker_orleans-client_1   tail -f /dev/null   Up            
+orleansdocker_orleans-silo_1     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_10    tail -f /dev/null   Up            
+orleansdocker_orleans-silo_11    tail -f /dev/null   Up            
+orleansdocker_orleans-silo_12    tail -f /dev/null   Up            
+orleansdocker_orleans-silo_13    tail -f /dev/null   Up            
+orleansdocker_orleans-silo_14    tail -f /dev/null   Up            
+orleansdocker_orleans-silo_15    tail -f /dev/null   Up            
+orleansdocker_orleans-silo_2     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_3     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_4     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_5     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_6     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_7     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_8     tail -f /dev/null   Up            
+orleansdocker_orleans-silo_9     tail -f /dev/null   Up 
+```
+
+> **Note**: The `Command` column on those examples are showing the `tail` command just because we are using the debugger container. If we were in production, it would be showing `dotnet OrleansSilo.dll` for example.
+
+# Docker Swarm
+
+Docker clustering stack is called **Swarm** and you can find more by reading its [documentation here](https://docs.docker.com/engine/swarm/).
+
+To run this article in a `Swarm` cluster, you don't have any extra work. When you run `docker-compose up -d` in a `Swarm` node, it will schedule containers based on the configured rules. The same applies to other Swarm-based services like [Docker Datacenter](https://www.docker.com/enterprise-edition), [Azure ACS](https://azure.microsoft.com/en-us/services/container-service/) (in Swarm mode), [AWS ECS Container Service](https://aws.amazon.com/ecs/) and so on. All you need to do is to deploy your `Swarm` cluster before deploy your **dockerized** Orleans application.
```

Comment:
Yeah, like I mentioned in your previously comment. It must have `stack` support on the engine. I'll add a comment there for it now. Thanks